Youtube Task/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html5>
+<html>
+	<head>
+        <title>Youtube Page</title>
+        <link rel = "icon" href =  "https://upload.wikimedia.org/wikipedia/commons/thumb/1/1b/Youtube_Music_logo.svg/1200px-Youtube_Music_logo.svg.png" type = "image/x-icon">
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="https://fonts.googleapis.com/css?family=Nunito+Sans&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <script src="Script.js"></script>
```

Comment:
Please add script.js just before closing the body tag

Reply:
done